### PR TITLE
Add deploy scripts for ExchangeIssuanceZeroEx

### PR DESCRIPTION
**Pull request by cgewecke**
Adds Optimism deployment script for index-coop-smart-contracts [ExchangeIssuanceZeroEx][2] 

Also deployed & verified on Kovan here: [0xfCCcD52248B906DB250705EF887e857Aa88fa36E][1]

[1]: https://kovan-optimistic.etherscan.io/address/0xfCCcD52248B906DB250705EF887e857Aa88fa36E#code
[2]: https://github.com/IndexCoop/index-coop-smart-contracts/blob/master/contracts/exchangeIssuance/ExchangeIssuanceZeroEx.sol

### CODE REVIEW

File: optimism/deploy/002_exchangeIssuanceZeroEx.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const zeroExExchangeAddress = await findDependency(ZERO_EX_EXCHANGE);
+
+  await deployExchangeIssuanceZeroEx(
+    CONTRACT_NAMES.EXCHANGE_ISSUANCE_ZEROEX,
+    wethAddress,
+    controllerAddress,
+    zeroExExchangeAddress,
+  );
+
+  async function deployExchangeIssuanceZeroEx(
```
Comment (dalberto) [inline optimism/deploy/002_exchangeIssuanceZeroEx.ts:42]: Minor nit, but if this is just an async function we wait for, why not just drop `async`, and call the function normally?

Comment (cgewecke) [reply optimism/deploy/002_exchangeIssuanceZeroEx.ts:42]: Yes, agree that definitely makes sense here.

**Review by dalberto — APPROVED** [review]

**Pull request merged**